src/ObjCRuntime/CpuArchitecture.cs

```diff
@@ -0,0 +1,15 @@
+namespace ObjCRuntime
+{
```

Comment:
Minor: but wrong coding guideline (should be same line). That's for this entire PR so it sounds like your settings are incorrect (:

Reply:
Clarification: for at least your `enum` and `struct`, those should be same line.

Reply:
Fixed.